lib/rack/builder.rb

```diff
@@ -31,6 +38,30 @@ module Rack
   # You can use +map+ to construct a Rack::URLMap in a convenient way.
 
   class Builder
+    # Config stores settings on what the server supports, such as whether it
+    # is multithreaded.
+    class Config
+      def initialize(multithread: true, concurrent: multithread)
+        @multithread = multithread
+        @concurrent = concurrent
+      end
+
+      # Whether the application server will invoke the application from multiple threads. Generally implies {concurrent?}.
+      def multithread?; @multithread; end
```

Comment:
This is fine, but is there a reason why we don't just use `attribution :multithread`?

Reply:
We'd have to alias it to `multithread?` (notice the `?`).  That doesn't seem worth it.  This isn't performance sensitive, it's only called during application setup.